src/cron/schedule-code-review.js

```diff
@@ -26,11 +26,51 @@ const getMappedStatusText = (key) => {
   return statusText;
 };
 
+/**
+ * Processes threads in a given channel for a specific status.
+ * @param {Channel} channel - Discord text channel.
+ * @param {string} statusText - Text to search in thread names.
+ * @param {object} statusConfig - Configuration object.
+ */
+const processChannelThreads = async (channel, statusText, statusConfig) => {
+  try {
+    const threads = await channel.threads.fetchActive();
+    const pendingThreads = threads.threads.filter((thread) =>
```

Comment:
I think now it process various threads and not only pending threads, what about calling it:

```
matchedThreads
```

or similar?